docs/source/development/traffic_ops.rst

```diff
@@ -229,30 +229,12 @@ To install the Traffic Ops Developer environment:
 	.. seealso:: `PostgreSQL instructions on setting up a database <https://wiki.postgresql.org/wiki/First_steps>`_.
 
 
-#. Use the ``setup`` command of the ``admin.pl`` script (see `Database Management`_ for usage) to set up the traffic_ops database(s).
-
-	 Example Output::
-
-		$ db/admin.pl --env=development setup
-		Using database.conf: conf/development/database.conf
-		Using database.conf: conf/development/database.conf
-		Using database.conf: conf/development/database.conf
-		Using database.conf: conf/development/database.conf
-		Using database.conf: conf/development/database.conf
-		Using database.conf: conf/development/database.conf
-		Executing 'drop database to_development'
-		Executing 'create database to_development'
-		Creating database tables...
-		Warning: Using a password on the command line interface can be insecure.
-		Migrating database...
-		goose: migrating db environment 'development', current version: 0, target: 20150210100000
-		OK    20141222103718_extension.sql
-		OK    20150108100000_add_job_deliveryservice.sql
-		OK    20150205100000_cg_location.sql
-		OK    20150209100000_cran_to_asn.sql
-		OK    20150210100000_ds_keyinfo.sql
-		Seeding database...
-		Warning: Using a password on the command line interface can be insecure.
+#. Use the ``reset`` and ``upgrade`` commands of the ``admin`` binary (see `Database Management`_ for usage) to set up the traffic_ops database(s).
+
+	.. code-block::shell
```

Comment:
This is why you're not seeing this in the output. It's also why that Perl snippet isn't being rendered. `..` in RST starts a comment - as you probably know. Without the space between the `::` and the syntax name, the lexer detects that as just a plain word, not a directive. It needs to match `^\s*.. [\w\-]+:: .*$` to be considered a directive.

tl;dr: you're missing a space after `::` and before `shell`.